src/lib/seam/components/DeviceDetails/NoiseSensorDeviceDetails.tsx

```diff
@@ -8,7 +9,11 @@ import { DeviceImage } from 'lib/ui/device/DeviceImage.js'
 import { NoiseLevelStatus } from 'lib/ui/device/NoiseLevelStatus.js'
 import { OnlineStatus } from 'lib/ui/device/OnlineStatus.js'
 import { ContentHeader } from 'lib/ui/layout/ContentHeader.js'
+import { NoiseSensorActivityList } from 'lib/ui/noise-sensor/NoiseSensorActivityList.js'
 import { NoiseThresholdsList } from 'lib/ui/noise-sensor/NoiseThresholdsList.js'
+import { TabSet } from 'lib/ui/TabSet.js'
+
+type TabType = 'Details' | 'Activity'
```

Comment:
The tab type should not depend on the "text" rendered for the tab.

```suggestion
type TabType = 'details' | 'activity'
```